src/statusUpdates.js

```diff
@@ -0,0 +1,12 @@
+// statusUpdates.js
+export function toggleTaskCompleted(task, populateTodoList, tasks, todoList) {
```

Comment:
- [ ] This function is missing something, when I check ✔️  as task as completed, normally it should add a line through the task as well, but in your case, it does not do that, please fix it, take a look at [this](https://www.youtube.com/watch?v=AcUd-_Yjjqg) reference video for more.

YOURS
 
<img width="822" alt="2" src="https://github.com/rudicarrilloypr/todo-list/assets/77843050/e96866b1-ec46-42a0-9c63-42b77f88d1bc">

ORIGINAL 

<img width="608" alt="1" src="https://github.com/rudicarrilloypr/todo-list/assets/77843050/96bff03a-49e1-400d-8e34-b96b692bff42">

Reply:
@joskalenda but that is not required on the project requirements of this part of the project